packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_home/views/home_page.dart

```diff
@@ -22,7 +26,17 @@ class HomePage extends StatelessWidget {
     final list = navItemsList(context);
     GoRouter router = GoRouter.of(context);
     return Scaffold(
-      body: child,
+      body: Column(
```

Comment:
Instead of using the Column->Expanded approach you can use BlocListener:child approach